MangoAPI.Client/src/app/interceptors/request-header.interceptor.ts

```diff
@@ -11,11 +12,17 @@ import { TokensService } from '../services/messenger/tokens.service';
 
 @Injectable()
 export class RequestHeaderInterceptor implements HttpInterceptor {
-  constructor(private tokensService: TokensService) {}
+  constructor(
+    private tokensService: TokensService,
+    private router: Router) {}
 
   intercept(request: HttpRequest<unknown>, next: HttpHandler): Observable<HttpEvent<unknown>> {
     const accessToken = this.tokensService.getTokens()?.accessToken;
 
+    if (accessToken === undefined) {
```

Comment:
Are you sure that this if is necessary? As I see, user in any case will be redirected to login if refresh interceptor fails

Reply:
why not?